src/profile-editor.h

```diff
@@ -29,6 +29,9 @@ void terminal_profile_edit (TerminalProfile *profile,
                             GtkWindow       *transient_parent,
                             const char      *widget_name);
 
+static gboolean terminal_profile_edit_dialog_page_scroll_event_cb (GtkWidget        *notebook,
+                                                                   GdkEventScroll   *event);
+
```

Comment:
Remove `terminal_profile_edit_dialog_page_scroll_event_cb` from .h file since this function is used in only one .c file.